packages/clerk-js/src/ui/contexts/components/SignIn.ts

```diff
@@ -121,16 +120,22 @@ export const useSignInContext = (): SignInContextType => {
 
   const signUpContinueUrl = buildURL({ base: signUpUrl, hashPath: '/continue' }, { stringify: true });
 
-  const sessionTaskUrl = buildSessionTaskRedirectUrl({
-    task: clerk.session?.currentTask,
-    path: ctx.path,
-    routing: ctx.routing,
-    baseUrl: signInUrl,
-    taskUrls: clerk.__internal_getOption('taskUrls'),
-  });
+  const onPendingSession: SetActiveNavigate = async ({ session }) => {
```

Comment:
This is currently not dealing with active sessions, therefore it pretty much broke all of the E2E tests... I'm raising a fix in a few.